packages/manager/modules/pci/src/projects/project/workflow/add/schedule/cron-validator.js

```diff
@@ -0,0 +1,159 @@
+import map from 'lodash/map';
+
+export default class CronValidator {
```

Comment:
There's already a validator existit in `web-universe-components` I think.
this logic is already complicated, we should probably try to not duplicate it

Reply:
you mean, code to validate UNIX Cron pattern is already exists in web-universe-components?

Reply:
https://github.com/ovh-ux/ng-ovh-web-universe-components/tree/master/src/cron-validator
but this implies adding it as a dependency to pci module

Reply:
can web-universe-components be added as a dependency to cloud? also I cant use it as it is. I need to make some changes to the code.

Reply:
We will benefit of not maintaining the same logic in multiple place.

I agreed with @marie-j's comment.

@radireddy Is that required a lot of change to the actual `WucCronValidator` service to fit your needs?

Reply:
Yes, lot of changes and testing is required. Can I address this as part of a separate issue?

Reply:
@radireddy Yes for sure.